docs/.dates_cache.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Pretty sure that this file should not be committed. Please remove.

Reply:
@heurtematte ^^

Reply:
done. Sorry I miss your ping